great_expectations/util.py

```diff
@@ -932,7 +932,7 @@ def lint_code(code: str) -> str:
 
 
 def filter_properties_dict(
-    properties: dict,
+    properties: Optional[dict] = None,
     keep_fields: Optional[Set[str]] = None,
     delete_fields: Optional[Set[str]] = None,
     clean_nulls: Optional[bool] = True,
```

Comment:
Could we remove `Optional` from all these boolean values that have default values? I believe `Optional` refers to the possibility of `None` (which is probably not intended?)

Reply:
I'm specifically referring to `clean_nulls`, `clean_falsy`, `keep_falsy_numerics`, and `inplace`

Reply:
Thanks for the clarification on this @cdkini .. I had totally flipped it in my mind